R/utils_insert_db.R

```diff
@@ -208,9 +208,14 @@ insert_riskmetric_to_db <- function(pkg_name,
       TRUE ~ as.character(riskmetric_assess[[metric$name]][[1]][1:length(riskmetric_assess[[metric$name]])])
     )
     
+    metric_score <- case_when(
+      is.na(riskmetric_score[[metric$name]][1]) ~ "NA",
+      TRUE ~ as.character(riskmetric_score[[metric$name]][1]) # rounding
```

Comment:
I see that you have a comment about rounding, why didn't you go ahead and round this value? Especially since you are storing it as a character.

Reply:
I think it would be better to store the full number (I actually don't like that we store a rounded version of the overall package score). I'm curious as to why we would store it as a character value though. It seems like it would be easy enough to store an actual number if we don't want to round.

Reply:
yes, I think I was having trouble storing the "NA" values, so I took a shortcut. My SQL skills are getting rusty. To do it the right way, wouldn't I have to store an numeric `NA` as `NULL` in the SQL table? And then when I pull the `NULL` back into R, I'd have re-convert it to a `NA_real_`...